python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -1766,13 +1803,15 @@ def _remeasure_star_background(self, result, background_to_photometric_ratio=Non
         # using an adaptive threshold detection iteration to ensure a
         # "Goldilocks Zone" for the fraction of detected pixels.
         backgroundOrig = result.background.clone()
-        median_background = np.median(backgroundOrig.getImage().array)
-        self.log.warning("Original median_background = %.2f", median_background)
+        median_background_orig = np.median(backgroundOrig.getImage().array)
+        self.log.info("Original median_background = %.2f", median_background_orig)
```

Comment:
Thank you for converting to an info on this one!  Also ... have we checked at how long this operation takes?  In my experience `background.getImage()` is surprisingly slow.  (Not to solve today; but something to consider for the future).

Reply:
Ok.  On the TODO list...